### PR TITLE
Workaround for posting many comments

### LINKED ISSUE #264: Lookout fails posting more than 30 comments
Lookout fails posting comments when they are "too much"

While working with [style-analyzer/typos_checker](https://github.com/irinakhismatullina/style-analyzer/tree/feature/typos-analyzer/lookout/style/typos_checker) and analyzing a PR with some different files, I got up to 39 messages from the analyzer, and then lookout failed posting the comments.

### PR DESCRIPTION
**Pull request by smacker**
Fix #264

### CODE REVIEW

File: provider/github/poster.go [hunk 1]
```diff
@@ -95,14 +102,51 @@ func (p *Poster) postPR(ctx context.Context, e *lookout.ReviewEvent,
...
+		_, resp, err = client.PullRequests.CreateReview(ctx, owner, repo, pr, req)
+		if err = p.handleAPIError(resp, err); err != nil {
+			return err
+		}
 	}
 
 	return nil
 }
 
+func splitReview(review *github.PullRequestReviewRequest, n int) []*github.PullRequestReviewRequest {
```
[the remark this answers is not in the document]
Comment (smacker) [reply provider/github/poster.go]: Looks like deployment is broken and staging still runs old lookout with bug:
```
$ kubectl get pods | grep lookout
lookout-4102562204-jc9dm                               1/1       Running            0          2d
```

[the remark this answers is not in the document]
Comment (bzz) [reply provider/github/poster.go]: is it tracked under https://github.com/src-d/issues-infrastructure/issues/230 ?

Comment (bzz) [inline provider/github/poster.go]: I wonder, how does this function behave when `len(review.Comments) == 92`.

A small comment with a description of how comments are spliced between multiple reviews would really help here.

[the remark this answers is not in the document]
Comment (smacker) [reply provider/github/poster.go]: yes. all good now

Comment (smacker) [reply provider/github/poster.go]: not sure I understand your question. It's internal function to split 1 review to more if there are more than N comments. Isn't it obvious from the signature?
```
func splitReview(review *github.PullRequestReviewRequest, n int) []*github.PullRequestReviewRequest
```

`len(review.Comments) == 92` will behave depend on N.

**Comment by smacker** [comment]
I'm merging now. But let me know if you need any comments clarification. I can't think about any which wouldn't just repeat the signature of the function.

File: provider/github/poster.go (hunk 1 above)
Comment (dpordomingo) [reply provider/github/poster.go]: I didn't understand the question neither. I found it properly coded and the tests explained it's behaviour too.

**Pull request merged**